platform/spire/src/access.py

```diff
@@ -86,13 +89,21 @@ def renew_ssh_cert() -> str:
 
 
 def refresh_cert(key_path, cert_path, ca_path, variant, ca_key_name, ca_cert_name):
-    if configuration.get_config().is_kerberos_enabled():
-        print("rotating", variant, "certs via keyreq")
-        if ca_path is None:
-            call_keyreq(variant + "-cert", key_path, cert_path)
-        else:
-            call_keyreq(variant + "-cert", key_path, cert_path, ca_path)
-    else:
+    errs = []
+
+    try:
+        if configuration.get_config().is_kerberos_enabled():
+            print("rotating", variant, "certs via keyreq")
+            if ca_path is None:
+                call_keyreq(variant + "-cert", key_path, cert_path)
+            else:
+                call_keyreq(variant + "-cert", key_path, cert_path, ca_path)
+            return
+    except Exception as e:
+        print("[keyreq failed]")
+        errs.append(e)
```

Comment:
This appears to cause errors to not be explained if keyreq fails but cert bypass succeeds, which would make it hard to debug a keyreq failure, because no details would be outputted besides `[keyreq failed]`. Is there any way that this can be easier? (Like a `--debug` or `--explain` option or something?)